### PR TITLE
Sort by country name, legend position fixes, refactor forecast layouts

### PR DESCRIPTION
**Pull request by areeh**
Sorted plots and traces by full country name
Moved legend to top and removed title from the combined forecast plot, updated axis labels to not lose information
Cleaned up the forecast plotting to use the same layout functions as the other plots
Fix header change in the files from ECDC (quick and dirty fix)

### CODE REVIEW

File: COVID_19_model.ipynb [hunk 1]
```diff
@@ -1669,7 +1733,7 @@
       "source": [
         "#@title Optional: generate forecasts and store the plots for all countries\n",
         "generate_forecast_plots = True #@param [\"False\", \"True\"] {type:\"raw\"}\n",
-        "download_forecast_plots = False #@param [\"False\", \"True\"] {type:\"raw\"}\n",
+        "download_forecast_plots = True #@param [\"False\", \"True\"] {type:\"raw\"}\n",
```
Comment (ttvand) [inline COVID_19_model.ipynb]: Maybe change this to False?

File: COVID_19_model.ipynb [hunk 2]
```diff
@@ -1229,11 +1336,11 @@
         "colab": {}
       },
       "source": [
-        "#@title Optionally download all html plots\n",
-        "download_plotly_plots = True #@param [\"False\", \"True\"] {type:\"raw\"}\n",
+        "#@title Optionally download html EDA plots\n",
+        "download_data_plots = True #@param [\"False\", \"True\"] {type:\"raw\"}\n",
```
Comment (ttvand) [inline COVID_19_model.ipynb]: Change this to False by default? It's a bit weird to have a Colab download 100 files by default as a non Intelecy user.

**Review by ttvand — APPROVED** [review]
Amazing work. Everything we discussed is in there and more.

File: COVID_19_model.ipynb [hunk 3]
```diff
@@ -72,6 +72,7 @@
       "source": [
         "#@title Imports\n",
         "import datetime\n",
+        "from google.colab import files\n",
```
Comment (harahu) [inline COVID_19_model.ipynb]: Double import

File: COVID_19_model.ipynb (hunk 1 above)
Comment (areeh) [reply COVID_19_model.ipynb]: Agreed, fixed

File: COVID_19_model.ipynb (hunk 2 above)
Comment (areeh) [reply COVID_19_model.ipynb]: Yeah definitely

File: COVID_19_model.ipynb (hunk 3 above)
Comment (areeh) [reply COVID_19_model.ipynb]: fixed

**Pull request merged**